build.gradle

```diff
@@ -30,6 +30,7 @@ buildscript {
 plugins {
     id 'io.github.gradle-nexus.publish-plugin' version '1.1.0'
     id 'org.jetbrains.dokka' version '1.7.10'
+    id 'org.jetbrains.kotlin.android' version '1.8.10' apply false
```

Comment:
The updates to gradle files were all auto-generated by the IDE when adding a compose activity. There is a separate ticket to update all gradle dependencies for browser switch v3 where this will be cleaned up and aligned with core dependencies